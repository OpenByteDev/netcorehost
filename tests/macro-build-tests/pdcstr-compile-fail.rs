fn main() {
    // with internal nul 
    let _ = netcorehost::pdcstr!("\0");
    let _ = netcorehost::pdcstr!("somerandomteststring\0");
    let _ = netcorehost::pdcstr!("somerandomteststring\0somerandomteststring");
    let _ = netcorehost::pdcstr!("somerandomteststring\0somerandomteststring");
}
