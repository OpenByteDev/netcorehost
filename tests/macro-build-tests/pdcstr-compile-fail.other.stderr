error: nul byte found in the literal
 --> $DIR/pdcstr-compile-fail.rs:3:34
  |
3 |     let _ = netcorehost::pdcstr!("\0");
  |                                  ^^^^

error: nul byte found in the literal
 --> $DIR/pdcstr-compile-fail.rs:4:34
  |
4 |     let _ = netcorehost::pdcstr!("somerandomteststring\0");
  |                                  ^^^^^^^^^^^^^^^^^^^^^^^^

error: nul byte found in the literal
 --> $DIR/pdcstr-compile-fail.rs:5:34
  |
5 |     let _ = netcorehost::pdcstr!("somerandomteststring\0somerandomteststring");
  |                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: nul byte found in the literal
 --> $DIR/pdcstr-compile-fail.rs:6:34
  |
6 |     let _ = netcorehost::pdcstr!("somerandomteststring\0somerandomteststring");
  |                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
